src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+package productDependency;
+
+import baseClassPackage.BasePage;
+import productDependency.CreateDependentCategoryPage;
+
+import java.io.IOException;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.testng.Assert;
+import utilPackages.JavaScriptExec;
+import utilPackages.PropertyValExtractors;
+import utilPackages.WaitClass;
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateDependentCategoryPage extends BasePage {
+	
+	/**
+	 * This constructor is used to initialize the webdriver in BasePage class, if
+	 * the user does not want to use page factory then this will take care of
+	 * initialization of the driver
+	 *
+	 * @param webdriver
+	 */
+	public CreateDependentCategoryPage(WebDriver webdriver) {
+		super(webdriver);
+	}
+
+	Logger log = Logger.getLogger(CreateDependentCategoryPage.class);
+	CreateDependentCategoryPage dependentCategoryPage;
+	PropertyValExtractors p = new PropertyValExtractors();
+	String sheetName = "AddProductCategory1";
+	String xlsxName = "/ProductDependency_TestData.xlsx";
+	
+	@FindBy(how = How.XPATH, using = "//input[@name='j_username']")
+	private WebElement enterLoginID;
+
+	/**
+	 * Method to enter login ID.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterLoginID() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+
+		log.info("Verifying the Login ID is available or not");
+		WaitClass.WaitForElementisDisplay(driver, 5, enterLoginID);
+		Assert.assertTrue(enterLoginID.isDisplayed());
```

Comment:
@mayankchhajer Remove this Wait , it will take extra time which is not necessary  here.

Reply:
Done